src/controllers/Trips.js

```diff
@@ -19,9 +52,23 @@ class Trips {
  * @returns {object} Details of booked trips
  * @memberof Trips
  */
-  static async multiCityRequest(req, res, next) {
+  async multiCityRequest(req, res, next) {
```

Comment:
I noticed you changed the method to a non-static one and added a constructor to this controller. This controller should be for handling Trips related requests, not initializing `sockets.io`.